.gitignore

```diff
@@ -0,0 +1 @@
+hammerspoon/hyper-apps.lua
```

Comment:
This will still shows up as modified, even though it's ignored:

```
$ git status -s 
 M hammerspoon/hyper-apps.lua
```

Is there a way to have the defaults still defined in `hyper.lua`, and if this file exists, load it and merge the defaults?